WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -723,16 +739,32 @@ private void updatePublishDateTextView() {
             return;
         }
         PostModel postModel = getPost();
-        if (PostUtils.shouldPublishImmediately(postModel)) {
-            mPublishDateTextView.setText(R.string.immediately);
-        } else {
-            String dateCreated = postModel.getDateCreated();
-            if (!TextUtils.isEmpty(dateCreated)) {
-                String formattedDate = DateUtils.formatDateTime(getActivity(),
-                                                                DateTimeUtils.timestampFromIso8601Millis(dateCreated),
-                                                                getDateTimeFlags());
-                mPublishDateTextView.setText(formattedDate);
+        String labelToUse = "%s"; // default label to use is just the formatted date
```

Comment:
Did you maybe forgot to remove or change the `"%s"` since that's not a valid string by itself? I think we can simply remove the initialization as it should be initialized later unless we have a good default value.